### PR TITLE
engine: fix conversion of RangeProperties → SizeProperties

### PR DESCRIPTION
**Pull request by kennytm**
## What have you changed? (mandatory)

Fix the wrong RangeProperties → SizeProperties conversion introduced in #20.

The "size" in RangeProperties is a cumulative size instead of actual range size. The overestimation caused "split" to spit out extremely small ranges and slow down import.

## What are the type of the changes? (mandatory)

- Bug fix (change which fixes an issue)

## How has this PR been tested? (mandatory)

`cargo test`

## Does this PR affect TiDB Lightning? (mandatory)

No

### CODE REVIEW

**Comment by kennytm** [comment]
PTAL @overvenus @3pointer cc @siddontang

**Review by overvenus — APPROVED** [review]

**Comment by kennytm** [comment]
PTAL @5kbpers

File: src/import/engine.rs [hunk 1]
```diff
@@ -121,16 +121,18 @@ impl Engine {
...
-                        offset: 0,
+                        size: range.size - prev_size,
+                        offset: range.size,
                     },
-                )
-            }));
+                );
+                prev_size = range.size;
+            }
+            res.total_size += prev_size;
```
Comment (5kbpers) [inline src/import/engine.rs]: If `range.size` is a cumulative size, I guess `res.total_size` should equal to `range.size`.

**Review by 5kbpers — APPROVED** [review]
LGTM

**Pull request merged**